gobblin-core/src/main/java/gobblin/writer/AvroHdfsDataWriter.java

```diff
@@ -11,8 +11,14 @@
 
 package gobblin.writer;
 
+import com.google.common.base.Preconditions;
```

Comment:
Oh okay. @zliu41 said they should be alphabetical, so I've been changing them as such.